nixos/doc/manual/installation/installing.xml

```diff
@@ -7,11 +7,11 @@
  <section xml:id="sec-installation-booting">
   <title>Booting the system</title>
 
-  <para>
-   NixOS can be installed on BIOS or UEFI systems. The procedure for a UEFI
-   installation is by and large the same as a BIOS installation. The
-   differences are mentioned in the steps that follow.
-  </para>
+  <para>NixOS can be installed on systems using UEFI or MBR bios. The procedure for a UEFI
```

Comment:
MBR bios isn't a thing. The names are *UEFI boot mode*, and *Legacy BIOS boot mode*.

I think that the first introduction of the terms should use the "boot mode" suffixes, but the following uses can be abbreviated to UEFI and Legacy without much fear.

This is an important distinction since MBR is a partition scheme *which happens* to have a span reserved for a boot loader, but many Legacy BIOS boot systems can boot from a GPT partitioned system, and conversely some, if not most, UEFI systems can boot a disk partitioned as MBR.

> The UEFI specification requires MBR partition tables to be fully supported.[1] However, some UEFI implementations immediately switch to the BIOS-based CSM booting upon detecting certain types of partition table on the boot disk, effectively preventing UEFI booting to be performed from EFI system partitions contained on MBR-partitioned disks.[4] 

 — https://en.wikipedia.org/wiki/EFI_system_partition

Reply:
Good point, I will update it to what you specified